lib/matplotlib/figure.py

```diff
@@ -1812,6 +1812,13 @@ def savefig(self, fname, *, frameon=None, transparent=None, **kwargs):
             The resolution in dots per inch.  If *None*, defaults to
             :rc:`savefig.dpi`.  If 'figure', uses the figure's dpi value.
 
+        quality : [ *None* | 95 >= scalar >= 1 ]
```

Comment:
1 <= scalar <= 95 seems to be the more natural way to do so?
I would also just leave the upper bound be 100 here and have the note about values above 95 below.

Reply:
thanks @anntzer. Yep, 1 <= scalar <= 95 does feel more natural.